clients/da-vinci-client/src/main/java/com/linkedin/davinci/ingestion/main/MainIngestionRequestClient.java

```diff
@@ -40,7 +39,7 @@
 public class MainIngestionRequestClient implements Closeable {
   private static final Logger LOGGER = LogManager.getLogger(MainIngestionRequestClient.class);
   private static final int REQUEST_MAX_ATTEMPT = 10;
-  private static final int HEARTBEAT_REQUEST_TIMEOUT_MS = 10 * Time.MS_PER_SECOND;
+  private static final int PERIODIC_REQUEST_TIMEOUT_SECONDS = 5;
```

Comment:
Nit:
Can we rename it to be `PERIODIC_HEART_BEAT_REQUEST_TIMEOUT_SECONDS` to be more specific?

Reply:
Got it.